cinema/serializers.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class CinemaSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=50)
+    description = serializers.CharField()
+    duration = serializers.IntegerField()
```

Comment:
you can use Meta class and `fields = '__all__'` attribute, no need to list each field separately